Source/Core/DolphinQt2/GameList/GameFileCache.cpp

```diff
@@ -22,22 +22,16 @@ GameFileCache::GameFileCache()
 
 bool GameFileCache::IsCached(const QString& path)
```

Comment:
This can now be a const-qualified member function as well. Ditto below for `GetFile()` as well.

(should have just started a review in retrospect, sorry for all the notifications).